config/.oxrc

```diff
@@ -59,16 +71,13 @@ event_mapping = {
         editor:save_as()
     end,
     ["ctrl_a"] = function()
```

Comment:
Perhaps save all should become Ctrl + Shift + A instead as Ctrl + A is now being used to select all